modules/persistent-storage-csi-efs-metrics-overview.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-csi-aws-efs.adoc
+// 
+
+[id="efs-metrics-overview_{context}"]
+= Usage metrics overview
+
+Amazon Web Services (AWS) Elastic File Service (EFS) usage metrics allows you to monitor how much EFS space is used by {product-title}.
+
```

Comment:
We need to enable the feature gate to use efs metrics. 
    enabled:
    - name: AWSEFSDriverVolumeMetrics

Reply:
@chao007 How do you do this? My understanding is that you do this by changing featureSet = TechPreviewNoUpgrade in the FeatureGate YAML file.  As per our docs Nodes > Working with Clusters > Enabling features using FeatureGates. 

I tried deleting AWSEFSDriverVolumeMetrics from the "disabled" to the "enabled" column in the YAML file, but his didn't work.

Reply:
@lpettyjo Please wait for this update. 
I see @gcharot  prefer this feature as GA.

Reply:
Yes waiting for https://github.com/openshift/api/pull/2013 to confirm if this is going to be TP or not

Reply:
Question on hold until TP vs GA status is resolved.

Reply:
Treating this as GA until told otherwise.

Reply:
Finalized as GA.